Assets/Mirror/Authenticators/BasicAuthenticator.cs

```diff
@@ -74,8 +74,8 @@ public void OnAuthRequestMessage(NetworkConnection conn, AuthRequestMessage msg)
 
                 conn.Send(authResponseMessage);
 
-                // Invoke the event to complete a successful authentication
-                OnServerAuthenticated.Invoke(conn);
+                // Accept the successful authentication
+                base.ServerAccept(conn);
```

Comment:
I dont think we need `base.` here

Reply:
Technically we don't but I thought it would be instructive to have that so it's obvious to the dev where that method is

Reply:
Fixed via commit